src/foam/nanos/auth/CapabilityAuthService.js

```diff
@@ -101,16 +101,24 @@ foam.CLASS({
       `,
       javaCode: `
         PM pm = PM.create(getX(), this.getClass(), "check");
+        Logger logger = (Logger) x.get("logger");
```

Comment:
So you are aware
```suggestion
        Logger logger = Loggers.logger(x, this);
```
will setup the logger so you don't have to continually call 
logger.debug(getClass().getSimpleName(), ..